src/main/scala/org/renci/cam/Utilities.scala

```diff
@@ -27,6 +30,30 @@ object Utilities {
           .collect {
             case (key, Right(value)) => (key, value)
           }
-    } yield curies
+    } yield PrefixesMap(curies)
+
+  def getBiolinkPrefixesFromFile: ZIO[Any, Throwable, PrefixesMap] =
+    for {
+      prefixesStr <- Task.effect(Source.fromFile("prefixes.json")).bracketAuto(source => Task.effect(source.mkString))
```

Comment:
`Source.fromFile("prefixes.json")` won't work once the application is packaged. Use something like `getClass().getResourceAsStream` instead.